controllers/operatorpolicy_status.go

```diff
@@ -693,6 +700,13 @@ func installPlanUpgradeCond(versions []string, approvableIPs []unstructured.Unst
 		cond.Message += " but multiple of those match the versions specified in the policy"
 	}
 
+	if complianceConfig == "NonCompliant" {
```

Comment:
Optional: It doesn't really matter in this case because there are only two values, but I prefer checking specifically for "Compliant" and if it's not, assume it's "NonCompliant". If we add additional options in the future, this is clearer.